language/Portuguese_Brazil/strings.po

```diff
@@ -1727,7 +1727,7 @@ msgstr "MAMEinfo"
 
 #: src/frontend/mame/ui/selmenu.cpp:132
 msgid "MARPScore"
-msgstr ""
+msgstr "MARPScore"
```

Comment:
I have no idea what is the semantics of this string. Can anyone clarify this? I'd advise adding a source-code comment so that other translators have a clue when translating this string.